notifier.rst

```diff
@@ -912,3 +912,41 @@ is dispatched. Listeners receive a
 ..    - Describe notification_on_failed_messages integration
 
 .. _`RFC 3986`: https://www.ietf.org/rfc/rfc3986.txt
+.. _`AllMySms`: https://github.com/symfony/symfony/blob/{version}/src/Symfony/Component/Notifier/Bridge/AllMySms/README.md
```

Comment:
Should we use alphabetical order ?

It's usage order for now

Reply:
Lets use alphabetical here (including RFC 3986), its easier to maintain

Reply:
Done :smile: